composer/models/resnet/model.py

```diff
@@ -76,9 +82,22 @@ def composer_resnet(model_name: str,
     if initializers is None:
         initializers = []
 
+    # Configure pretrained/weights based on torchvision version
+    if pretrained:
+        weights = 'IMAGENET1K_V2'
+        warnings.warn(
+            'The ``pretrained`` argument for composer_resnet is deprecated and will be removed in the future. Please use ``weights`` instead.'
+        )
+    if weights:
+        pretrained = True
```

Comment:
Can we raise an exception if both are specified (i.e. pretrained is True and weights is not None)?